bve/src/parse/kvp/mod.rs

```diff
@@ -0,0 +1,265 @@
+//! Generic parser for key-value pair format. Not quite toml, not quite INI.
+//!
+//! No frills format. Does not deal with casing, comments, etc. That must be
+//! dealt with ahead of time. This just deserializes the file as is. However
+//! whitespace is trimmed off the edges of values
+//!
+//! There may be arbitrary duplicates.
+//!
+//! The first section before a section header is always the unnamed section `None`.
+//! This differs from an empty section name `Some("")`
```

Comment:
Plz no

Reply:
Then how else am I supposed to represent `[]` as a section name. Remember I'm just deserializing this, I don't actually care about the meaning of anything.